tools/getkeys/index.js

```diff
@@ -112,24 +151,33 @@ async function getClient() {
 }
 
 (async () => {
-    const rcP = rcTools.loadRC();
-    const clientP = getClient();
-    const [client, rc] = await Promise.all([clientP, rcP]);
+    const rc = await rcTools.loadRC();
 
     if (rc.secrets === undefined) {
         rc.secrets = {};
     }
 
-    // Primary key vault for test/dev secrets shared by Microsoft-internal teams working on FF
-    await getKeys(client, rc, "prague-key-vault");
+    // For debugging, change the following to 'false' to skip connecting to
+    // Azure Key Vault and instead use the secrets cached in '~/.fluidtoolrc'.
+    if (true) {
+        const client = await getClient();
+
+        // Primary key vault for test/dev secrets shared by Microsoft-internal teams working on FF
+        await getKeys(client, rc, "prague-key-vault");
 
-    try {
-        // Key Vault with restricted access for the FF dev team only
-        await getKeys(client, rc, "ff-internal-dev-secrets");
-        console.log("Overrode defaults with values from the FF internal keyvault.");
-    } catch (e) { }
+        try {
+            // Key Vault with restricted access for the FF dev team only
+            await getKeys(client, rc, "ff-internal-dev-secrets");
+            console.log("\nNote: Default dev/test secrets overwritten with values from internal key vault.");
+        } catch (e) { }
+    }
+    
+    console.log(`\nWriting '${path.join(os.homedir(), ".fluidtoolrc")}'.`);
     await rcTools.saveRC(rc);
+    await saveEnv(rc.secrets);
```

Comment:
Previously, the secrets were persisted to the user's environment as they were individually fetched.  We now write them in bulk as the last step.